tests/smoke/smoketests.sh

```diff
@@ -198,6 +198,20 @@ for BACKEND in ${BACKENDS[@]}; do
 			fi
 			echo " "
 
+			echo ">>>      [x]           [ ]       Tests grb::Launcher on a K-core decomposition on the dataset"
+			echo "                                 EPA.mtx. The launcher is used in automatic"
+			echo "                                 mode and the IO mode is sequential in direct mode."
+			echo "                                 Launcher::exec is used with statically sized input and"
+			echo "                                 statically sized output."
+			echo "Functional test executable: ${TEST_BIN_DIR}/kcore_decomposition_${BACKEND}"
+			if [ -f ${INPUT_DIR}/EPA.mtx ]; then
+				$runner ${TEST_BIN_DIR}/kcore_decomposition_${BACKEND} ${INPUT_DIR}/EPA.mtx direct 1 1 &> ${TEST_OUT_DIR}/kcore_decomposition_${BACKEND}_EPA_${P}_${T}.log
+				grep 'Test OK' ${TEST_OUT_DIR}/kcore_decomposition_${BACKEND}_EPA_${P}_${T}.log || printf 'Test FAILED.\n'
```

Comment:
@anhansson By selecting EDA.mtx, do we already have a checksum perhaps, e.g., the coreness level is known? If yes, it seems the test does not check for it, is that correct (or does it happen inside the `.cpp`, e.g.?)

Reply:
(Offline communication from just now: the functional test was done manually, and is not automatic. I have added a warning to the function as a result, and opened an issue for it)